homeassistant/components/screenlogic/climate.py

```diff
@@ -138,8 +138,8 @@ def hvac_action(self) -> HVACAction:
     def preset_mode(self) -> str:
         """Return current/last preset mode."""
         if self.hvac_mode == HVACMode.OFF:
-            return HEAT_MODE(self._last_preset).title
-        return HEAT_MODE(self.entity_data[VALUE.HEAT_MODE][ATTR.VALUE]).title
+            return HEAT_MODE(self._last_preset).name.lower()
+        return HEAT_MODE(self.entity_data[VALUE.HEAT_MODE][ATTR.VALUE]).name.lower()
```

Comment:
Making these lowercase is a breaking change. As is the same for making the options in the enum sensors lowercase. So I would suggest splitting this to a separate PR to keep it clear in the release notes

Reply:
Automations users have that use this. And it's not a bad change, but we need to properly mention it